src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -117,6 +117,19 @@ static bool _parseNumber(const char** content, float* number)
 }
 
 
+static bool _parseNumber(const char* content, char** end, float* number)
```

Comment:
Hmm... this looks can be merged with the above... Actually it should be.

Reply:
I updated it